calculator/main.py

```diff
@@ -13,7 +13,19 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
+
+    my_operations={}
```

Comment:
You could also use deepcopy if you want to make a new copy of the dict in this function

    my_operations = copy.deepcopy(operations)